evm/contracts/apps/Base.sol

```diff
@@ -38,13 +39,12 @@ abstract contract IBCAppBase is IIBCModule {
      * NOTE: You should apply an `onlyIBC` modifier to the function if a derived contract overrides it.
      */
     function onChanOpenInit(
-        IbcCoreChannelV1GlobalEnums.Order,
-        string[] calldata connectionHops,
-        string calldata portId,
-        string calldata channelId,
-        IbcCoreChannelV1Counterparty.Data calldata counterpartyEndpoint,
-        string calldata version,
-        address relayer
+        IBCChannelOrder,
+        uint32,
+        uint32,
+        IBCChannelCounterparty calldata,
+        bytes32,
+        address
```

Comment:
ik the linter says to remove the names but we should figure out how to configure it to keep them. unnamed params on interfaces are kinda useless as documentation

Reply:
it's not the interface but the abstract contract, the `IIBCModule` interface have all parameters